modules.d/00systemd/module-setup.sh

```diff
@@ -13,7 +13,7 @@ check() {
 
 # called by dracut
 depends() {
-    return 0
+    echo base
```

Comment:
@PrincessDoreena the suggestion from @jozzsi makes sense for me. This change should be in the `systemd-initrd` dracut module instead. Would you be able to make this change in this PR or should someone else take it over from here ?